org.eclipse.xtext.tests/src/org/eclipse/xtext/nodemodel/BasicNodeIterable.java

```diff
@@ -0,0 +1,21 @@
+/*******************************************************************************
+ * Copyright (c) 2020 itemis AG (http://www.itemis.eu) and others.
```

Comment:
The copyright header should list your name I suppose

Reply:
"(c) 2020 Robert Lewis" or "(c) 2020 Robert Lewis and others"?
Given the work I'm building on here, it could be argued this isn't 100% my work.

Reply:
we ususally do the and others pattern